src/components/AuthForm.tsx

```diff
@@ -22,17 +21,11 @@ const AuthForm: React.FC<LoginFieldProps> = ({
   const [email, setEmail] = useState<string>('');
   const [password, setPassword] = useState<string>('');
 
-  const navigation = useNavigation();
-
-  const { state, signin } = useContext(AuthContext);
+  const { signin } = useContext(AuthContext);
```

Comment:
I'd still like to use the state to determine if the app should navigate to the next "Signed-In" pages. Will add this back.

Reply:
Ah, Already handled in App.tsx.